crates/stages/src/stage_metrics.rs

```diff
@@ -0,0 +1,35 @@
+use metrics::Counter;
+use reth_interfaces::p2p::error::DownloadError;
+use reth_metrics::Metrics;
+use std::fmt;
+
+/// Stagedsync header metrics
+#[derive(Metrics)]
+#[scope("stages.headers")]
+pub struct HeaderMetrics {
+    /// Number of headers successfully retrieved
```

Comment:
I'm not sure how hard it would be, but could we use these comments as the help text unless overriden? This is sort of how Clap does it, dedupes some code. Unless there are objections to it 😄

Reply:
that's actually really easy! i thought of this as well, but was wary of introducing implicit behavior. will make this default and properly document it